helix-core/src/test/java/org/apache/helix/TestListenerCallbackBatchMode.java

```diff
@@ -119,6 +119,9 @@ public void beforeClass()
   @AfterClass
   public void afterClass()
       throws Exception {
+    String testClassName = this.getShortClassName();
```

Comment:
Do we need one in BeforeClass?  If we need this being print out before and after class methods, should we put this into ZkTestBase (like what we do in for beforeTest and afterTest in ZkTestBase)?

Reply:
Let me move this log to ZkTestBase